assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -213,6 +228,51 @@ const ChecklistTab = (props) => {
           }
           ariaLabel={TEXT.RECOMMENDED_TITLE}
         >
+          {isVideoOptimizationEnabled && !areVideosInitiallyOptimized.current && (
```

Comment:
what do you think about separating this out into its own component? This checklistTab is so big.